integration/kubernetes/helm-chart/alluxio/Chart.yaml

```diff
@@ -10,7 +10,6 @@
 #
 
 name: alluxio
-apiVersion: 2.2.0-SNAPSHOT
```

Comment:
Helm will actually complain on this `apiVersion` for not being `v1` and helm commands will fail working on this repo. By the key `apiVersion` it really expects an K8s api version. The Alluxio version is fed with `.Values.imageTag` so we don't actually need this field in `Chart.yaml`. If we really need this field we should then rename it.